javascript/javascript1/week1/housey_pricey.js

```diff
@@ -0,0 +1,25 @@
+let width = parseInt(prompt('Enter width of the property'));
+let deep = parseInt(prompt('Enter depth of the property'));
+let high = parseInt(prompt('Enter height of the property'));
+let gardenSizeInM2 = parseInt(prompt('Enter size of garden of the property'));
+let price = parseInt(prompt('Enter your house price'));
+let volumeInMeters = width * deep * high;
+let housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+
+
+function housey_pricy () {
```

Comment:
It's absolutely a very good move to use function here and make your code reusable, BUT you should try to avoid to use any variables outside of your function, and even if you need to get information from outside of the function, you are supposed to get them through the arguments of your function

Reply:
I have updated housey pricey homework :)